shared/yarn.lock

```diff
@@ -10716,6 +10723,11 @@ mksnapshot@^0.3.4:
     fs-extra "0.26.7"
     request "2.x"
 
+moment@*:
```

Comment:
Maybe this is accidental?

Reply:
I don't know how to properly add dependencies it seems.

Reply:
I removed it and ran `yarn` and it got added again. It looks like this is because of the `@types/moment` dependency. We have other `@*` types in this file too.